content/en/docs/gsoc/_index.md

```diff
@@ -73,7 +73,7 @@ Ideas
 
 ### Media sharing
 
-One of the nice features of Subsonic is its [ability to generate a sharing link](http://www.subsonic.org/pages/features.jsp) for a track/album/artist/playlist to send to friends, so that they can listen/download the music without having an account on your instance. This is a nice replacement to alternative to youtube links to share music.
+One of the nice features of Subsonic is its [ability to generate a sharing link](http://www.subsonic.org/pages/features.jsp) for a track/album/artist/playlist to send to friends, so that they can listen/download the music without having an account on your instance. This is a nice replacement to an alternative to youtube links to share music.
```

Comment:
I think @jvoisin meant "This is a nice alternative to youtube links to share music." here

Reply:
Yeah, I also had the same thought.

Reply:
Yup, I confirm. Please change it to something like `This is a nice replacement for youtube links to share music.`